src/components/Settings/index.tsx

```diff
@@ -59,9 +59,9 @@ const MenuFlyout = styled(AutoColumn)`
   padding: 16px;
 `
 
-const ExpandColumn = styled(AutoColumn)`
+const ExpandColumn = styled(AutoColumn)<{ padTop: boolean }>`
```

Comment:
is this generating a warning in the console about passing unrecognized props to the DOM?

Reply:
`$padTop`

Reply:
Use `$padTop` as suggested to avoid console warnings:

```suggestion
const ExpandColumn = styled(AutoColumn)<{ $padTop: boolean }>`
```